bin/heatmap.py

```diff
@@ -0,0 +1,107 @@
+import sys
```

Comment:
I don't think uploading a large ckpt file to github is a good idea, i suggest you train from scratch to get your testing  model, it will take no more than half a day.